plugins/search-backend-module-pg/src/PgSearchEngine/PgSearchEngine.ts

```diff
@@ -92,6 +93,12 @@ export class PgSearchEngine implements SearchEngine {
       this.databaseStore.query(tx, pgQuery),
     );
 
+    if (pgQuery?.types && !rows.length) {
```

Comment:
This seems to now throw the `MissingIndexError` if a search query happens to return an empty result set (e.g. searching a valid, existing index for the term `ThisAbsolutelyDoesNotExist`)

Maybe we just don't support this `MissingIndexError` for postgres?

Reply:
As we export the `MissingIndexError` now maybe its ok to leave Postgres out of this for now and it can be implemented if needed in the future. The issue was opened specifically for elastic search I think.